paddle/fluid/operators/send_op.cc

```diff
@@ -45,18 +46,24 @@ class SendOp : public framework::OperatorBase {
     distributed::RPCClient* rpc_client =
         distributed::RPCClient::GetInstance<RPCCLIENT_T>();
 
+    std::shared_ptr<framework::BlockingQueue<int>> ret_q = nullptr;
```

Comment:
Confusion about why we using a BlockingQueue in each RPC call. maybe we only need to process a request handler for one RPC call.

Reply:
Thanks!Done!